worlds/metroidprime/docs/en_Metroid Prime.md

```diff
@@ -0,0 +1,89 @@
+# Metroid Prime Archipelago
+
+An Archipelago implementation of Metroid Prime multiworld randomizer using [randomprime](https://github.com/randovania/randomprime/)
+
+## Setup Guide
+
+To get started or for troubleshooting, see [the Setup Guide](./docs/setup_en.md).

```

Comment:
Oh right, forgot to mention that this link does not work on webhost (the ./docs/setup one)